### ISSUE TITLE
Plugins use full 100% of CPU core in headless mode

### ISSUE THREAD

**Issue by ryze312** [sadko4u/lsp-plugins#261]
All plugins use 100% of a single core while running in headless mode.
Everything else works as expected.

Usage with UI:
```
> ps -p 5122 -o %cpu,cmd
%CPU CMD
 6.2 lsp-plugins-para-equalizer-x32-stereo -c main.cfg
```
Usage in headless mode:
```
> ps -p 5021 -o %cpu,cmd
%CPU CMD
 100 lsp-plugins-para-equalizer-x32-stereo -hl -c main.cfg
```
Version: 1.2.2
OS: Artix Linux OpenRC
CPU: Intel Core i7-3612QM

**Comment by sadko4u** [comment]
Thank you, didn't notice that before.
This is a bug, will be fixed in 1.2.3.

**Comment by sadko4u** [comment]
There was an improper implementation of the `sleep_msec` function in the `lsp-runtime-lib` module.
The fix can be applied with this patch: https://github.com/lsp-plugins/lsp-runtime-lib/commit/dbc16bcfb151c0b1d24f40e51316f8e3f34bb7da

**Comment by sadko4u** [comment]
Available in 1.2.3 release!
